From: Marcus Better <marcus@better.se>
Subject: do not build input-send and input-recv

These programs have limited usefulness and have been left out.

---
 GNUmakefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/GNUmakefile b/GNUmakefile
index 73bae9d..947e015 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -6,7 +6,7 @@ include mk/Variables.mk
 CFLAGS	+= -DVERSION='"$(VERSION)"'
 
 # build
-TARGETS	:= lsinput input-events input-kbd input-send input-recv lircd.conf
+TARGETS	:= lsinput input-events input-kbd lircd.conf
 HEADERS	:= EV.h REL.h ABS.h MSC.h LED.h SND.h REP.h KEY.h BTN.h BUS.h SW.h
 
 # default target
@@ -44,7 +44,7 @@ input.o: input.c $(HEADERS)
 
 install: build
 	$(INSTALL_DIR) $(bindir) $(mandir)/man8
-	$(INSTALL_BINARY) lsinput input-events input-kbd input-send input-recv $(bindir)
+	$(INSTALL_BINARY) lsinput input-events input-kbd $(bindir)
 	$(INSTALL_DATA) lsinput.man $(mandir)/man8/lsinput.8
 	$(INSTALL_DATA) input-kbd.man $(mandir)/man8/input-kbd.8
 	$(INSTALL_DATA) input-events.man $(mandir)/man8/input-events.8
-- 
tg: (a20db48..) debian/remove-input-send-recv (depends on: master features/exclude-libm)
